openmc/mgxs/mgxs.py

```diff
@@ -1530,21 +1532,21 @@ def rxn_rate_tally(self):
         return self._rxn_rate_tally
 
 
-class NuFissionXS(MGXS):
-    """A fission production multi-group cross section."""
+class FissionXSBase(MGXS):
```

Comment:
There is a `FissionXS` class - any reason you chose not to sublcass it from `FissionXSBase`? If that was simply neglected, no need to refactor it as a subclass of `FissionXSBase` just yet...